### PR TITLE
Tabs: Allow tabs to change tabs

### PR DESCRIPTION
**Pull request by AlexGStapleton**
This PR will allow the user to tab through the tabs. This PR is for accessibility reasons.

### CODE REVIEW

File: widgets/tabs/js/tabs.js [hunk 1]
```diff
@@ -52,7 +52,7 @@ jQuery( function ( $ ) {
 				}
 			};
 			
-			$tabs.click( function () {
+			$tabs.on('keyup click', function () {
 				selectTab( this );
```
Comment (braamgenis) [inline widgets/tabs/js/tabs.js]: This automatically selects the tab as soon as the Tab button is released, which isn't usual Tab button behavior. Pressing Tab should cycle focus to the tab and then it should only be selected when the Enter key is pressed.

Comment (braamgenis) [inline widgets/tabs/js/tabs.js]: This automatically selects the tab as soon as the Tab button is released, which isn't usual Tab button behavior. Pressing Tab should cycle focus to the tab and then it should only be selected when the Enter key is pressed.

Comment (AlexGStapleton) [reply widgets/tabs/js/tabs.js]: [It looks like both methods are accepted](https://www.w3.org/TR/wai-aria-practices/#tabpanel), but displaying the contents automatically upon focus is the general recommendation if there's no latency delay during the tab change (as in no requests to the server in general) ([source](https://www.w3.org/TR/wai-aria-practices/#kbd_selection_follows_focus))

Regardless, there are a few other recommendations (such as the tabindex and keyboard control) that should be made.

Comment (braamgenis) [reply widgets/tabs/js/tabs.js]: Oh, that's interesting. In my general experience tab components use manual activation, at least, I can't recall seeing one use automatic activation. Anyway, all good then. 👍 

>Regardless, there are a few other recommendations (such as the tabindex and keyboard control) that should be made.

Not sure what you mean by this last bit? Recommendations? Was there something else you still wanted to change or add?

Comment (AlexGStapleton) [reply widgets/tabs/js/tabs.js]: The tabindex needs to only be set on the active tab and a tabindex must be set for the tab content.

You navigate through the tabs using arrows.

I would try [the example tab](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-1/tabs.html) for how the tab is ideally meant to work. Thoughts?

Comment (braamgenis) [reply widgets/tabs/js/tabs.js]: Ah, ok. Yeah, I agree, let's go with the recommended functionality for better accessibility. 👍

**Comment by AlexGStapleton** [comment]
How does that look?

File: widgets/tabs/tpl/default.php [hunk 2]
```diff
@@ -27,7 +28,7 @@
 	<div class="sow-tabs-panel-container">
 	<?php foreach ( $tabs as $i => $tab ) : ?>
 		<div class="sow-tabs-panel">
-			<div class="sow-tabs-panel-content">
+			<div class="sow-tabs-panel-content" role="tabpanel" <?php echo $i != $initial_tab_index ? 'aria-hidden="true"' : ''; ?>>
```
Comment (braamgenis) [inline widgets/tabs/tpl/default.php]: Doesn't this also need `tabindex=0` when selected? Or perhaps the parent container `sow-tabs-panel-container`?

**Comment by braamgenis** [comment]
All good! 👍

**Pull request merged**